wpilibc/src/main/native/cpp/logging/CSVLogCell.cpp

```diff
@@ -0,0 +1,38 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "frc/logging/CSVLogCell.h"
+
+#include <algorithm>
+#include <iostream>
```

Comment:
Use `<wpi/raw_ostream.h>` and `wpi::outs()` instead of `<iostream>` and `std::cout` for performance reasons.